### PR TITLE
Add rkt and k8s binaries

### PR DESCRIPTION
**Pull request by hectorj2f**
Add support to configure a rkt and kubectl version to be installed in a cluster.

Related to:  
https://github.com/giantswarm/yochu/issues/14
https://github.com/giantswarm/mayu/pull/38

ping @giantswarm/team-core

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://api.travis-ci.org/giantswarm/yochu.svg)](https://travis-ci.org/giantswarm/yochu) [![](https://godoc.org/github.com/giantswarm/yochu?status.svg)](http://godoc.org/github.com/giantswarm/yochu) [![IRC Channel](https://img.shields.io/badge/irc-%23giantswarm-blue.svg)](https://kiwiirc.com/client/irc.freenode.net/#giantswarm)
 
-`yochu` provisions already running CoreOS hosts with Docker, `etcd`, `fleet` and `iptables`.
+`yochu` provisions already running CoreOS hosts with Docker, `etcd`, `fleet`,`rkt` and `iptables`.
```
Comment (denderello) [inline README.md]: `k8s` is missing here.

**Comment by denderello** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ Our custom binaries can be found at:
 - etcdctl: https://downloads.giantswarm.io/etcd/v2.1.0-gs-1/etcdctl
 - fleet: https://downloads.giantswarm.io/fleet/v0.11.3-gs-2/fleetd
 - fleetctl: https://downloads.giantswarm.io/fleet/v0.11.3-gs-2/fleetctl
+- rkt: https://downloads.giantswarm.io/rkt/v1.1/rkt
+- kubectl: https://downloads.giantswarm.io/k8s/v1.1.8/kubectl
```
Comment (zeisss) [inline README.md]: Seeing this: Do we have some way for enduser to see what we actually serve in different versions?

File: steps/docker/docker.go [hunk 3]
```diff
@@ -46,8 +46,6 @@ var (
 	fileMode = os.FileMode(0755)
 	services = []string{serviceName, socketName}
 	paths    = []string{servicePath, socketPath}
-
```
Comment (zeisss) [inline steps/docker/docker.go]: why was this removed?

Comment (htr) [inline steps/docker/docker.go]: why was it there before?

Comment (zeisss) [inline steps/docker/docker.go]: See #10

Comment (hectorj2f) [inline steps/docker/docker.go]: @zeisss @htr PR in WIP

File: README.md (hunk 2 above)
Comment (hectorj2f) [inline README.md]: What do you mean here ? Do you mean If we add something atop of kubectl 1.1.8 to let end users to know it ?

Comment (zeisss) [inline README.md]: it was just a question I had and totally not relevant for the PR. Will we list every release we do of all the tools here? do we clean this up at some point? Will we do this manually?

Comment (hectorj2f) [inline README.md]: In my opinion, we could omit it, and just specify that these binaries are downloaded from an URL with the form: `https://downloads.giantswarm.io/{COMPONENT}/{VERSION}/{BINARY_NAME}`

File: steps/docker/docker.go (hunk 3 above)
Comment (hectorj2f) [inline steps/docker/docker.go]: @zeisss @htr reverted according to the upstream version

File: README.md (hunk 1 above)
Comment (hectorj2f) [inline README.md]: done, thx

File: README.md (hunk 2 above)
Comment (zeisss) [inline README.md]: How does the user then know which versions work? Or are we setting up an autobuild system to automatically build every release and push it?

Comment (hectorj2f) [inline README.md]: The valid versions are in the config.yaml.dist. Otherwise we could have them in a config file, but I don't see the need of specifying them in the README, IMHO. In other words, I don't see why to list them in the README as well.

Comment (zeisss) [inline README.md]: Then this PR should add them there?

**Comment by zeisss** [comment]
LGTM :heart:
